### PR TITLE
Google openid -  look up chache bean by name

### PR DESCRIPTION
**Pull request by taba90**
lookup of the openid auth cache by name and not by type to fix issue with logout.

### CODE REVIEW

File: src/web/app/src/main/resources/geostore-ovr.properties [hunk 1]
```diff
@@ -34,13 +34,13 @@
...
-# googleOAuth2Config.internalRedirectUri=
-# googleOAuth2Config.enabled=false
-# googleOAuth2Config.authenticatedDefaultRole=USER
+googleOAuth2Config.clientId=1009729363864-2i5coc4ef0fguoe0uqcfbuq7r5gno0fv.apps.googleusercontent.com
+googleOAuth2Config.clientSecret=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
+googleOAuth2Config.autoCreateUser=true
+googleOAuth2Config.redirectUri=http://localhost:9191/geostore/rest/openid/google/callback
+googleOAuth2Config.internalRedirectUri=http://localhost:9191/geostore/rest
+googleOAuth2Config.enabled=true
+googleOAuth2Config.authenticatedDefaultRole=USER
```
Comment (offtherailz) [inline src/web/app/src/main/resources/geostore-ovr.properties:43]: Committed in error

File: src/web/app/src/main/resources/geostore-ovr.properties [hunk 2]
```diff
@@ -50,7 +50,7 @@
 
 
 ## if not specified the optional properties  needs to be populated
-# googleOAuth2Config.discoveryUrl=https://accounts.google.com/.well-known/openid-configuration
+googleOAuth2Config.discoveryUrl=https://accounts.google.com/.well-known/openid-configuration
```
Comment (offtherailz) [inline src/web/app/src/main/resources/geostore-ovr.properties:53]: Did you committed this intentionally?

File: src/modules/rest/impl/src/main/java/it/geosolutions/geostore/services/rest/security/oauth2/openid/OpenIdSecurityConfiguration.java [hunk 3]
```diff
@@ -1,11 +0,0 @@
-package it.geosolutions.geostore.services.rest.security.oauth2.openid;
```
Comment (offtherailz) [inline src/modules/rest/impl/src/main/java/it/geosolutions/geostore/services/rest/security/oauth2/openid/OpenIdSecurityConfiguration.java:1]: Why did you removed all these openId classes?

**Review by offtherailz — CHANGES_REQUESTED** [review]
Please look at my reviews.

Did you tried the issues locally to verify they are all solved @taba90  ?

File: src/modules/rest/impl/src/main/java/it/geosolutions/geostore/services/rest/security/oauth2/openid/OpenIdSecurityConfiguration.java (hunk 3 above)
Comment (taba90) [reply src/modules/rest/impl/src/main/java/it/geosolutions/geostore/services/rest/security/oauth2/openid/OpenIdSecurityConfiguration.java:1]: Were a leftover and not actually used by any code involved by openid

**Review by offtherailz — APPROVED** [review]

**Comment by offtherailz** [comment]
@tdipisa @taba90 
After testing on master, we have to do an additional release of geostore 1.9.1.
This because the current MapStore 2022.02.xx has 1.9.0, so we have to do a new release or replace 1.9.0. and update mapstore version for geostore.
Master of MapStore is 1.9-SNAPSHOT.

**Pull request merged**